recipes/armadillo/all/conanfile.py

```diff
@@ -25,7 +28,9 @@ class ArmadilloConan(ConanFile):
         "mkl",
         "hdf5",
     )
-    settings = "os", "compiler", "build_type", "arch"
+
+    package_type = "library"
```

Comment:
Unnecessary as this was added in https://github.com/conan-io/conan-center-index/pull/17320